modules/simple_forms_api/app/models/simple_forms_api/vba_40_10007.rb

```diff
@@ -24,6 +24,11 @@ def metadata
       }
     end
 
+    def zip_code_is_us_based
+      # TODO: Implement this
+      true
+    end
```

Comment:
Will these methods be implemented in a future PR? If so, might I suggest creating a class that has this method defined and pointing this and all other classes utilizing this method to inherit from it, like so:

```ruby
class BaseVbaForm
  def zip_code_is_us_based
    true
  end
end
```

```ruby
module SimpleFormsApi
  class VBA210845 << BaseVbaForm
    ...
  end
end
```

Perhaps I'm overthinking things but any other shared logic can live there too. I'd love your thoughts on this.

Reply:
Oh yeah that's a great idea! I hesitate to create an abstraction out of this but maybe that day is soon approaching. I think for now I'll keep this as-is but maybe when the form models are more stable we can look at them and abstract some of their methods into a parent class.